spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/Bindable.java

```diff
@@ -50,11 +50,11 @@
 	/**
 	 * Enumerates all the input binding names.
 	 */
-	Set<String> getInputs();
+	Set<String> getInputNames();
```

Comment:
After a quick chat w/ @markfisher - whose initial comment triggered the renaming, we agreed on keeping the original names for consistency with the rest of the methods (will fix on merge).